docs/source/examples.rst

```diff
@@ -579,9 +579,9 @@ Special Camera Settings
            def param_gauss(u, v):
                x = u
                y = v
-               d = np.sqrt(x * x + y * y)
-               sigma, mu = 0.4, 0.0
-               z = np.exp(-((d - mu) ** 2 / (2.0 * sigma ** 2)))
+               sigma, mu = 0.4, [0.0, 0.0]
+               d = np.linalg.norm(np.array([x - mu[0], y - mu[1]]))
```

Comment:
sounds good, but why not something like
sigma, mu_x ,mu_y = 0.4, 0.0, 0.0
d =np.sqrt( (x - mu_x)**2+ (y -mu_y)**2)

Reply:
I think representing mu as a vector/coordinate makes more sense, because that's what it is.

Reply:
okay, then we can merge it like it is right now